src/basis/type/date.js

```diff
@@ -93,17 +93,7 @@ function dateTransform(defaultValue, nullable){
   return transform;
 }
 
-var defValue = new Date(0);
```

Comment:
I know, but Object.freeze does not work actually (at least, at Chrome) and tests showed problems with using Proxies with Dates. So Proxy should be removed anyway. And Object.freeze, I think, should be removed too, because it does not work